### PR TITLE
honeyview: Add 64bit architecture

### PR DESCRIPTION
**Pull request by dodorz**

### CODE REVIEW

File: bucket/honeyview.json [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
         "identifier": "Freeware",
         "url": "https://www.bandisoft.com/honeyview/eula/"
     },
-    "url": "https://dl.bandisoft.com/honeyview/HONEYVIEW-PORTABLE.ZIP",
-    "homepage": "https://www.bandisoft.com/honeyview/",
-    "hash": "1c6f6184838bf6f62ebcf994c2ec17dbe9e5f111672eb988f6b48d80dc11e392",
-    "bin": "Honeyview32.exe",
+    "architecture":{
+        "64bit":{
+            "url": "https://dl.bandisoft.com/honeyview/HONEYVIEW-SETUP.EXE#/dl.7z",
```
Comment (Ash258) [inline bucket/honeyview.json]: This file contains both architectures.

First one (smaller) is 32 bit and second is 64bit.

image

Comment (dodorz) [reply bucket/honeyview.json]: Yes, but I don't known how to extract the file with same names separately in powershell.

Comment (Ash258) [reply bucket/honeyview.json]: I will provide you fixed manifest later.

**Comment by Ash258** [comment]
There is `config.ini`. Is persisting needed?

**Comment by dodorz** [comment]
Thanks

**Pull request merged**